### PR TITLE
chore: test failure fix for appengine standard storage

### PR DESCRIPTION
**Pull request by bajajneha27**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] **Tests** pass
- [x] **Lint** pass: `bundle exec rubocop`
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: appengine/standard-storage/Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ gem "sinatra", "~>2.1"
 # [END gae_standard_storage_dependencies]
 
 group :test do
-  gem "capybara"
+  gem "capybara", "~>3.38.0"
```
Comment (dazuma) [inline appengine/standard-storage/Gemfile:23]: Do you mean to pin to 3.38.x or is 3.39+ also okay?

**Review by dazuma — APPROVED** [review]

File: appengine/standard-storage/Gemfile (hunk 1 above)
Comment (bajajneha27) [reply appengine/standard-storage/Gemfile:23]: `3.38.0` has the changes we need. I meant anything above or equal to `3.38.0`

**Comment by bajajneha27** [comment]
The CI is failing because we need Ruby 2.7 for Capybara 3.38.0 onwards. Ruby 2.6 is still the oldest Ruby version we support, as of now.

File: appengine/standard-storage/Gemfile (hunk 1 above)
Comment (dazuma) [reply appengine/standard-storage/Gemfile:23]: So maybe better `~> 3.38` so that it will also allow 3.39.x, 3.40.x, etc.

**Pull request merged**